frontend/yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@apidevtools/json-schema-ref-parser@9.0.9":
   version "9.0.9"
-  resolved "https://registry.yarnpkg.com/@apidevtools/json-schema-ref-parser/-/json-schema-ref-parser-9.0.9.tgz#d720f9256e3609621280584f2b47ae165359268b"
+  resolved "https://registry.npmjs.org/@apidevtools/json-schema-ref-parser/-/json-schema-ref-parser-9.0.9.tgz"
```

Comment:
You probably should not have these changes. Did you run `npm install` instead of `yarn install` ?